add: log likelihood and readme (#4)
AlissonRP/erfG: readme.md

## Introduction

A project to simplify the discovering of maximum likelihood points by applying 
the  `erf` function. 

## Installation

```r
devtools::install_github("https://github.com/AlissonRP/erfG")

```

## Project

Project developed in the course of Statistics-UFSM for the conclusion of the 

> course. Based in a arecent paper, Zea and Andrade (2021) proposed a new generator

Reviewer: ```suggestion
course. Based in a recent paper, Zea and Andrade (2021) proposed a new generator
```

Author: I just copy that part of the Profesor, I couldn't imagine there would be errors. Thanks friennd.

of distributions called the erf-$G$ family. Their proposal provides special 
distributions without adding complexity to parametric spaces of resulting models.
Empirical evidences shows that the proposed family may solve issues of flat or 

> quasi-red likelihoods in some baselines.Due to great applied potential of the 

Reviewer: ```suggestion
quasi-red likelihoods in some baselines. Due to the great applied potential of the 
```

> new model, this package provides a computional implementatio of the erf-$G$ 

Reviewer: ```suggestion
new model, this package provides a computational implementation of the erf-$G$ 
```

> family. The density, distribution ande quantile functions,  besides functions to

Reviewer: ```suggestion
family. The density, distribution and quantile functions,  besides functions to
```
